bundles/org.openhab.binding.homeconnect/README.md

```diff
@@ -161,15 +161,60 @@ The channel parameter uses the following syntax: `homeconnect:<thing type id>:<b
+String                 Washer_laundry_care_washer_spin_speed              "Spin speed"                         {channel="homeconnect:washer:api_bridge_at_home:washer1:laundry_care_washer_spin_speed"}
+String                 Washer_laundry_care_washer_idos1_level             "i-Dos 1 Level"                      {channel="homeconnect:washer:api_bridge_at_home:washer1:laundry_care_washer_idos1_level"}
+String                 Washer_laundry_care_washer_idos2_level             "i-Dos 2 Level"                      {channel="homeconnect:washer:api_bridge_at_home:washer1:laundry_care_washer_idos2_level"}
+Switch                 Washer_laundry_care_washer_idos1                   "i-Dos 1 state"                      {channel="homeconnect:washer:api_bridge_at_home:washer1:laundry_care_washer_idos1"}
+Switch                 Washer_laundry_care_washer_idos2                   "i-Dos 2 state"                      {channel="homeconnect:washer:api_bridge_at_home:washer1:laundry_care_washer_idos2"}
+Switch                 Washer_laundry_care_washer_vario_perfect           "Vario Perfect state"                {channel="homeconnect:washer:api_bridge_at_home:washer1:laundry_care_washer_vario_perfect"}
+Switch                 Washer_laundry_care_washer_less_ironing            "Ironing state"                      {channel="homeconnect:washer:api_bridge_at_home:washer1:laundry_care_washer_less_ironing"}
+Switch                 Washer_laundry_care_washer_pre_wash                "Prewash state"                      {channel="homeconnect:washer:api_bridge_at_home:washer1:laundry_care_washer_pre_wash"}
+Switch                 Washer_laundry_care_washer_rinse_plus              "Amount additional rinses"           {channel="homeconnect:washer:api_bridge_at_home:washer1:laundry_care_washer_rinse_plus"}
+Switch                 Washer_laundry_care_washer_rinse_hold              "Spin hold"                          {channel="homeconnect:washer:api_bridge_at_home:washer1:laundry_care_washer_rinse_hold"}
+Switch                 Washer_laundry_care_washer_soak                    "Soaking state"                      {channel="homeconnect:washer:api_bridge_at_home:washer1:laundry_care_washer_soak"}
+Number:Mass            Washer_laundry_care_washer_load_recommendation     "Load recommendation"                {channel="homeconnect:washer:api_bridge_at_home:washer1:laundry_care_washer_load_recommendation"}
+Number:Dimensionless   Washer_program_energy                              "Estimated energy consumption"       {channel="homeconnect:washer:api_bridge_at_home:washer1:program_energy"}
+Number:Dimensionless   Washer_program_water                               "Estimated water consumption"        {channel="homeconnect:washer:api_bridge_at_home:washer1:program_water"}
+String                 Washer_basic_actions_state                         "Basic control"                      {channel="homeconnect:washer:api_bridge_at_home:washer1:basic_actions_state"}
```

Comment:
similar comments as previous, please use consistent naming strategy, so it is a true copy paste solution.